src/shared-components/alert/index.js

```diff
@@ -82,7 +83,7 @@ class Alert extends React.Component {
       >
         <AlertContentContainer>
           <Icon
-            css={[alertIconStyles]}
+            css={css`${alertIconStyles};`}
```

Comment:
Can you clarify something? `alertIconStyles` is created in style.js using `css`. Is there a difference between creating it with `css` vs wrapping in `css` here?

Reply:
oh nope, there's not. Both ways here are valid. https://emotion.sh/docs/composition

I just did a global search on `css=` and updated all of it to use the template literal.

Reply:
👍